MicrowaveGame/Assets/Resources/Prefabs/Projectiles/ProjectileBullet.prefab

```diff
@@ -113,6 +113,10 @@ PrefabInstance:
       propertyPath: m_LocalScale.y
       value: 0.1
       objectReference: {fileID: 0}
+    - target: {fileID: 7178730013344640899, guid: f53cd03f428072b4a9ad1ac029ec8236, type: 3}
+      propertyPath: m_LocalScale.z
+      value: 0
+      objectReference: {fileID: 0}
```

Comment:
Not sure why this was changed but transform scale z should not be 0, I would leave it the same as the other values (0.1) or 1.0